components/vf-collapse/README.md

```diff
@@ -4,7 +4,8 @@
 
 ## About
 
-Collapse and expand components are useful in a variety of utilitarian situations to show and hide information.
+This component has been <span style="color: rgb(228, 0, 70);">deprecated</span>. Please use the <a class="vf-link" href="https://stable.visual-framework.dev/components/vf-details">vf-details</a> component.
```

Comment:
Why you using inline css when we as library provide ton's of utility classes make use of them; use `vf-u-text-color--red--dark` or search CSS by this name `vf-u-text-color` plenty of colors available.

Reply:
This is done as per the guidelines at - https://stable.visual-framework.dev/guidance/deprecating-components/

Reply:
Changes done in latest commit.